api/controllers/game/stalemate.js

```diff
@@ -47,7 +47,7 @@ module.exports = async function (req, res) {
 
       await Game.updateOne({ id: gameId }).set({
         lastEvent: {
-          change: 'stalemateRequest',
+          change: 'stalemateAccept',
```

Comment:
Couldn't previously differentiate between stalemate was requested vs accepted in socket event via the `change` attribute. This makes the difference more explicit